Dockerfile

```diff
@@ -5,20 +5,20 @@ RUN apk add --no-cache \
     make \
     gcc \
     g++ \
-    git \
     python
 
 WORKDIR /opt/pizza-sync
 ADD . .
 
 # frontend
-RUN cd /opt/pizza-sync/frontend && yarn && yarn run build:prod && rm -rf node_modules
+RUN cd frontend && yarn && yarn run build:prod && rm -rf node_modules
 
 # backend
-RUN cd /opt/pizza-sync/backend/ && yarn && mv ../frontend/dist ./public
+RUN cd backend/ && yarn
+
+# insert compiled front in backend dir
+RUN mv frontend/dist backend/public
```

Comment:
I meant a copy while in the last stage, below, sorry I wasn't clear :)

What I had in mind is that copying this there makes it clear what are the two pieces of data needed to build the final image: the back and the front.
I'm a nitpicking, but aiming for perfection ;)